components/sup/protocols/event.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto2";
```

Comment:
Is proto3 a workable option for y'all in rust-land? A2 is all on proto3 and while there is interop between the two, the docs imply that it gets kinda weird at times.

Reply:
@danielsdeleo I think we should be able to use v3 for this, though historically we've stuck with v2. It's been a while, though, so I should dig back into that. Migrating the rest of our protobufs over to v3 might be a big shift, but since this bit is green field, I think the risks here are less.

I'll take a look at that and report back.

Reply:
@danielsdeleo Check out https://github.com/habitat-sh/habitat/pull/6344/commits/81e60d39abfdcebe42e683b3044f081fcb0bbc86